vulcanai2/models/basenetwork.py

```diff
@@ -281,6 +285,19 @@ def _create_network(self):
         :return: None
         """
         pass
+    
+    def forward(self, *inputs):
```

Comment:
I think I didn't know what was going on in this discussion. why is forward in base_network???

Reply:
the discussion is in our slack group